app/assets/stylesheets/components/navbar.scss

```diff
@@ -2,7 +2,7 @@
   max-height: 100%;
   margin-bottom: 50px;
   display: flex;
-  padding: 0 168px;
+  padding: 7px 168px;
```

Comment:
I can see the reason for including this, but the navbar was designed to be in that position at the top of the screen so I think we should leave this out of the PR.

Reply:
This is only relevant if remove the divider as the ruler. if we do the navbar sticks to the top of the screen that's why I needed to add some padding to the top to keep it in his original position.